packages/extension/src/ui/features/accountNfts/alephium-nft.service.ts

```diff
@@ -19,6 +19,22 @@ export const fetchCollectionAndNfts = async (
       parentAndTokenIds[collectionId] = [tokenId]
     }
   }
+
+  // TODO: Fix explorer backend for 000301 NFTs
+  if (nftMetadataz.length != nonFungibleTokenIds.length) {
```

Comment:
Question: If NFT `000301` is supported in the explorer backend, do we still need this?

Reply:
No, we don't need this after the explorer backend is fixed